package.json

```diff
@@ -35,6 +35,10 @@
     "./lib/slim": {
       "require": "./lib/slim.js",
       "import": "./lib/esm/slim.js"
+    },
+    "./lib/utils": {
+      "require": "./lib/utils.js",
```

Comment:
You’d need a default condition also, to support all the node versions enzyme supports.

Reply:
What would it catch? `"exports"` was added in Node.js 12.7 when this should always match the `"require"` case unless using modules.

Reply:
node 13.1-13.7, iirc.

Reply:
The other exports here would also fail if that was the case. Let's fix this once it actually becomes an issue for users.

Reply:
This one will continue to fail for enzyme because that *is* the case; this way we’d have to wait and fix them one by one, which is much more work and much slower.